Wendy/assignment1Answers.html

```diff
@@ -0,0 +1,219 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>JavaScript Immersion Assignment 1 || Wendy Goodwin</title>
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+	    <link href="css/main.css" rel="stylesheet">    
+	    <link href="css/mystyles.css" rel="stylesheet">
+	</head>
+
+	<body>
+
+		<div id="wrap">
+			<div class="title">
+				<h1> JavaScript Immersion Assignment I </h1>
+				<h3> by Wendy Goodwin </h3>	
+				<h4> 
+
+					<a href="https//github/portlandcodeschool/jse=fall15-1">	View assignment specifications</a>
+
+
+				</h4>
+			</div>
+			<div class="row" id="part1">
+				<h3> Part I </h3>
+				<div class="col-md-4">
```

Comment:
Nice work on your answers and explanations here. Take a look at the comments below and and the homework solutions for some more specific reasoning behind why things work the way they do.

I might also suggest that in future homeworks you keep solutions to a markdown format rather than an html format, for ease of readability and commenting. That said, I like the ambition!